avocado/utils/multipath.py

```diff
@@ -0,0 +1,151 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+"""
+Module with multipath related utility functions.
+It needs root access.
```

Comment:
So let's handle this properly. We have `avocado.utils.process.can_sudo()` which returns `True`/`False` and can be used to fail early. Something like:
```python
...
if not process.can_sudo():
    raise ImportError('This module requires superuser rights.')
...
```
Then you have to execute everything that applies with `sudo=True`.